### ISSUE TITLE
When I paste a large string with CTRL+V, page scrolling to down 

### ISSUE THREAD

**Issue by onursenos** [ckeditor/ckeditor4#5332]
## Type of report

Bug
 

### Expected result

I dont wanna scrolling page problem. My page scrolling when I paste a large string in a CKEDITOR textarea.

How can I fix that problem?
 
## Other details

* Browser: Chrome
 
* CKEditor version: 4

**Comment by taufik-nurrohman** [comment]
When you paste a large chunk of text in the comment form below, does it scroll down?

**Comment by onursenos** [comment]
no, in my laravel project. not on the github or the other sites

**Comment by taufik-nurrohman** [comment]
I&rsquo;m asking this because I think that&rsquo;s the expected behavior when someone pastes a large chunk of text.

**Comment by onursenos** [comment]
yes I know that, but I dont want this skill in my project. How can I disable that?

so much thank you.
